CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+## 0.86.0 / 2025-10-07
+
+> [!NOTE]
+> This release introduces the status subresource (behind the `StatusForConfigurationResources` feature gate) for `ServiceMonitor`, `PodMonitor` and `Scrapeconfig` custom resources. It is only supported for `Prometheus` resources.
+
+> [!IMPORTANT]
+> This release enables UTF-8 character support in label names, metric names and prometheus rule expressions for Prometheus 3.0+ compatibility.
+
+* [CHANGE] Remove automatic addition of the `metadata-wal-records` feature flag for Prometheus versions >= 3.4. #7893
+* [CHANGE] Add miscellaneous validations to the `ScrapeConfig` CRD. #7856 #7823 #7835 #7838 #7838 ##7966
+* [CHANGE/FEATURE] Add support for UTF-8 characters in label names and metric names in PrometheusRules and relabel configurations. #7637 #7985
```

Comment:
should we add another entry for https://github.com/prometheus-operator/prometheus-operator/pull/7637 but related to the admission webhook service option?

Reply:
ya could add in bugfix

Reply:
yes